docs/README.md

```diff
@@ -0,0 +1,110 @@
+## Controllers
+### Clustermesh Controller
+#### Description
+This is the controller responsible for managing the mesh between the Clusters. It watches `clusters.cluster.x-k8s.io` and create the necessary cloud resources to establish the mesh between them leveraging on Crossplane CRs. 
+
+#### How to use
```

Comment:
Since in the same document you talk about the clustermesh controller and the security controller, but one use the other, I think is good to state that for the clustermesh to work, it is not necessary to create a security group, as the controller will do it for you

Reply:
Do you mean during the "How to use" ? I believe it's confusing since it's not necessary to create the security group, why would we talk about the need not to in this part? 

It's very clear in [this](https://github.com/topfreegames/provider-crossplane/pull/54/files#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938R43) part that the clustermesh controller will handle the creation